packages/@react-spectrum/s2/intl/hr-HR.json

```
@@ -2,8 +2,9 @@
   "actionbar.actions": "Radnje",
   "actionbar.actionsAvailable": "Dostupne radnje.",
   "actionbar.clearSelection": "Poništi odabir",
-  "actionbar.selected": "Odabrano: { count }",
-  "actionbar.selectedAll": "Sve je odabrano",
+  "actionbar.selected": "{broj, množina, =0 {ništa nije odabrano} drugo {# odabrano}}",
```

I think this is supposed to be
```suggestion
  "actionbar.selected": "{count, plural, =0 {ništa nije odabrano} other {# odabrano}}",
```

---

Thanks for catching @snowystinger I've corrected it